test/ct-woodpecker-genissuer/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"bytes"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/base64"
+	"encoding/pem"
+	"fmt"
+	"math"
+	"math/big"
+	"time"
+)
+
+func main() {
+	issuerKey, _ := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
+	keyBytes, _ := x509.MarshalECPrivateKey(issuerKey)
+	encodedBytes := base64.StdEncoding.EncodeToString(keyBytes)
+	fmt.Printf("Key:\n%s\n", encodedBytes)
+
+	serial, _ := rand.Int(rand.Reader, big.NewInt(math.MaxInt64))
+
+	template := &x509.Certificate{
+		Subject: pkix.Name{
+			CommonName: "ct-woodpecker CA",
```

Comment:
Let's add a random suffix to this since different woodpecker instances may have different root CAs.